From: Mathias Krause <minipli@googlemail.com>
Date: Wed, 16 Jul 2014 19:43:08 +0200
Subject: asus-nb-wmi: Constify asus_quirks[] DMI table
Git-commit: 55d1e9d83dd7ba3a6f28241dfac90f07af10d4f4
Patch-mainline: v3.17
References: boo#911438

Constify the asus_quirks[] DMI table. There's no need to have it
writeable during runtime.

Signed-off-by: Mathias Krause <minipli@googlemail.com>
Cc: Corentin Chary <corentin.chary@gmail.com>
Signed-off-by: Matthew Garrett <matthew.garrett@nebula.com>
Acked-by: Jean Delvare <jdelvare@suse.de>
---
 drivers/platform/x86/asus-nb-wmi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/platform/x86/asus-nb-wmi.c
+++ b/drivers/platform/x86/asus-nb-wmi.c
@@ -84,7 +84,7 @@ static int dmi_matched(const struct dmi_
 	return 1;
 }
 
-static struct dmi_system_id asus_quirks[] = {
+static const struct dmi_system_id asus_quirks[] = {
 	{
 		.callback = dmi_matched,
 		.ident = "ASUSTeK COMPUTER INC. X401U",
